droplets_test.go

```diff
@@ -178,6 +178,110 @@ func TestDroplets_Destroy(t *testing.T) {
 	}
 }
 
+func TestDroplets_Kernels(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/droplets/12345/kernels", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Trying to understand the ask :) My understanding is that this API call lists *available* kernels and has nothing to do with the actual kernel currently set on the droplet (https://developers.digitalocean.com/documentation/v2/#list-all-available-kernels-for-a-droplet). What am I missing?